ccdproc/image_collection.py

```diff
@@ -112,6 +112,9 @@ def __init__(self, location=None, keywords=None, info_file=None,
         if keywords:
             self.keywords = keywords
 
+        if extension is not None:
```

Comment:
That should probably be replaced with:

```
if extension is None:
    self._extension = 0
else:
    self._extension = extension
```

That should fix the failing tests (I hope).